sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -138,6 +138,18 @@ Console.WriteLine(deletedCert.ScheduledPurgeDate);
 await client.PurgeDeletedCertificateAsync("MyCertificate");
 ```
 
+### List Certificates
+`GetCertificatesAsync` Enumerates the certificates in the vault, returning select properties of the
+certificate, sensative feilds of the certificate will not be returned. This operation
```

Comment:
"sensitive fields". Please be sure to run spell check. If doing this in Code, there are a few good extensions you can use.

Also, this line should be "certificate. Sensitive fields...". That's a new sentence.